source/services/ApplicationVersionService.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from "../helpers/ApiRequest";
+
+import VERSION_STATUS from "../types/VersionStatusTypes";
+
+interface GetApplicationVersionStatus {
+  status: VERSION_STATUS;
+  updateUrl: string;
+}
+const getApplicationVersionStatus = async (): Promise<
+  GetApplicationVersionStatus | Record<string, unknown>
+> => {
+  const response = await get("/version");
+
+  if (response.status !== 200) {
+    return {
+      // NOTE: we assume things are kind of great when AWS doesnt respond nice...
```

Comment:
This comment could be removed. If the intent is not clear enough I would suggest to extract the return into a variable:

```typescript
const fallbackResponse = { ... }

return fallbackResponse
```